src/wp-includes/class-wp-oembed.php

```diff
@@ -105,6 +105,7 @@ public function __construct() {
 			'#https?://([a-z]{2}|www)\.pinterest\.com(\.(au|mx))?/.*#i' => array( 'https://www.pinterest.com/oembed.json', true ),
 			'#https?://(www\.)?wolframcloud\.com/obj/.+#i' => array( 'https://www.wolframcloud.com/oembed', true ),
 			'#https?://pca\.st/.+#i'                       => array( 'https://pca.st/oembed.json', true ),
+			'#https?://datastudio\.google\.com/c/u/0/reporting/.+#i' => array( 'https://datastudio.google.com/oembed', true ),
```

Comment:
@swissspidy I know this is late but it occurred to me while prepping the release notes that the `/u/0/` section of the URL suggests the user URL rather than the sharing URL.

Signed in to a second account I see `/u/1` in my URLs.

When I select to share a report I get a URL in the format `https://datastudio.google.com/s/[A-Za-z0-9]+`

Reply:
Good catch. We should improve the regex here then.